### PR TITLE
Add release notes for 0.14.7

### LINKED ISSUE #1065: Add release notes for 0.14.7
Add release notes for 0.14.7

### PR DESCRIPTION
**Pull request by aswinsuryan**
Fixes: https://github.com/submariner-io/submariner-website/issues/1065

### CODE REVIEW

**Review by sridhargaddam — CHANGES_REQUESTED** [review]
These notes should first be added to the release branch and sync'ed from there.

File: src/content/community/releases/_index.en.md [hunk 1]
```diff
@@ -5,6 +5,15 @@ weight = 40
...
 
+## v0.14.7 (October 17, 2023)
+
+* Fixed an issue with iptables, where xtables.lock will be created as directory and subsequent invocations of iptables on the host
+  will fail.
+* IPsec connection will be now retried if there are any `whack` errors.
+* The Pluto process now waits for 1 minute before returning the error code `33`.
+* IP forwarding will be now enabled in Submariner interfaces if not enabled by default.
+* Submariner gateway now starts pluto process only when `libreswan` connects to first remote endpoint
+
```
Comment (tpantelis) [inline src/content/community/releases/_index.en.md:16]: Also need https://github.com/submariner-io/submariner/pull/2531:

"_The Submariner gateway now retries reading local node information on startup to reduce pod restarts if the Kubernetes API server is temporarily unavailable._"

and https://github.com/submariner-io/submariner/pull/2597:

"_Reduced data path downtime with Libreswan cable driver when gateway pod restarts._"

**Review by tpantelis — CHANGES_REQUESTED** [review]

**Comment by tpantelis** [comment]
> These notes should first be added to the release branch and sync'ed from there.

Instructions are [here](https://submariner.io/development/release-process/#release-notes-final-releases)

File: src/content/community/releases/_index.en.md (hunk 1 above)
Comment (tpantelis) [reply src/content/community/releases/_index.en.md:16]: The first one above was not release-noted in other branches so I assume was deemed as not needing it.
The others are already included in the release-0.14 branch. We just need the ones I noted above in the  release-0.14 branch: https://github.com/submariner-io/submariner-website/pull/1073

**Pull request closed without merging**